assets/app/actions/siteActions.js

```diff
@@ -65,7 +66,15 @@ export default {
 
   fetchSiteConfigsAndAssets(site) {
     return this.fetchSiteConfigs(site).then((site) => {
-      return this.fetchSiteAssets(site);
+      return this.fetchSiteAssets(site).then((site) => {
```

Comment:
Currently we make 4 (I think?) requests to GitHub to get various pieces of content for the site. One request should be able to get the `_config.yml`, `_navigation.json`, and general content of the site correct? I'm not sure when the user need access to the `assets` folder, but my feeling is that its necessary when they enter the editor or click the 'Media' link. Is that accurate?